cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -154,147 +86,138 @@ func getCertsSubCommands(defaultKubernetesVersion string) []*cobra.Command {
 	kubeadmscheme.Scheme.Default(cfg)
 
 	var cfgPath string
-	var subCmds []*cobra.Command
-
-	subCmdProperties := []struct {
-		use      string
-		short    string
-		long     string
-		examples string
-		cmdFunc  func(cfg *kubeadmapi.InitConfiguration) error
-	}{
-		{
-			use:      "all",
-			short:    "Generates all PKI assets necessary to establish the control plane",
-			long:     allCertsLongDesc,
-			examples: allCertsExample,
-			cmdFunc:  certsphase.CreatePKIAssets,
-		},
-		{
-			use:     "ca",
-			short:   "Generates a self-signed kubernetes CA to provision identities for components of the cluster",
-			long:    caCertLongDesc,
-			cmdFunc: certsphase.CreateCACertAndKeyFiles,
-		},
-		{
-			use:     "apiserver",
-			short:   "Generates an API server serving certificate and key",
-			long:    apiServerCertLongDesc,
-			cmdFunc: certsphase.CreateAPIServerCertAndKeyFiles,
-		},
-		{
-			use:     "apiserver-kubelet-client",
-			short:   "Generates a client certificate for the API server to connect to the kubelets securely",
-			long:    apiServerKubeletCertLongDesc,
-			cmdFunc: certsphase.CreateAPIServerKubeletClientCertAndKeyFiles,
-		},
-		{
-			use:     "etcd-ca",
-			short:   "Generates a self-signed CA to provision identities for etcd",
-			long:    etcdCaCertLongDesc,
-			cmdFunc: certsphase.CreateEtcdCACertAndKeyFiles,
-		},
-		{
-			use:     "etcd-server",
-			short:   "Generates an etcd serving certificate and key",
-			long:    etcdServerCertLongDesc,
-			cmdFunc: certsphase.CreateEtcdServerCertAndKeyFiles,
-		},
-		{
-			use:     "etcd-peer",
-			short:   "Generates an etcd peer certificate and key",
-			long:    etcdPeerCertLongDesc,
-			cmdFunc: certsphase.CreateEtcdPeerCertAndKeyFiles,
-		},
-		{
-			use:     "etcd-healthcheck-client",
-			short:   "Generates a client certificate for liveness probes to healthcheck etcd",
-			long:    etcdHealthcheckClientCertLongDesc,
-			cmdFunc: certsphase.CreateEtcdHealthcheckClientCertAndKeyFiles,
-		},
-		{
-			use:     "apiserver-etcd-client",
-			short:   "Generates a client certificate for the API server to connect to etcd securely",
-			long:    apiServerEtcdServerCertLongDesc,
-			cmdFunc: certsphase.CreateAPIServerEtcdClientCertAndKeyFiles,
-		},
-		{
-			use:     "sa",
-			short:   "Generates a private key for signing service account tokens along with its public key",
-			long:    saKeyLongDesc,
-			cmdFunc: certsphase.CreateServiceAccountKeyAndPublicKeyFiles,
-		},
-		{
-			use:     "front-proxy-ca",
-			short:   "Generates a front proxy CA certificate and key for a Kubernetes cluster",
-			long:    frontProxyCaCertLongDesc,
-			cmdFunc: certsphase.CreateFrontProxyCACertAndKeyFiles,
+
+	// Special case commands
+	// All runs CreatePKIAssets, which creates everything
```

Comment:
This seems redundant? I'd lean on the `Short` field defined below instead of a comment.